src/component/events.ts

```diff
@@ -31,17 +31,26 @@ export const storeEvents = mutation({
   },
   returns: v.null(),
   handler: async (ctx, { sdkKey, payloads, options }) => {
-    await handleScheduleProcessing(ctx, { sdkKey, options });
+    await handleScheduleProcessing(ctx, {
+      sdkKey,
+      options,
+    });
 
-    // @ts-expect-error Count is internal
-    const numEvents = await ctx.db.query("events").count();
-    if (numEvents + payloads.length > EVENT_CAPACITY) {
-      console.warn("LaunchDarkly event store is full, dropping event.");
+    const numEvents = (await ctx.db.query("events").collect()).length;
```

Comment:
convex-test can't use count. When we make EVENT_CAPACITY configurable, we'll need to make sure it doesn't exceed or get close to the max number of documents / bytes you can read in one udf.